server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/service/ExportService.java

```diff
@@ -84,25 +93,42 @@ public ExportService(MailboxBackup mailboxBackup, BlobStore blobStore, BlobExpor
                 .export()));
     }
 
-    private void deleteBlob(BlobId blobId) {
-        blobStore.delete(blobStore.getDefaultBucketName(), blobId)
+    private Mono<Void> deleteBlob(BlobId blobId) {
+        return Mono.defer(() -> blobStore.delete(blobStore.getDefaultBucketName(), blobId))
             .onErrorResume(e -> {
                 LOGGER.error("Error deleting Blob with blobId: {}", blobId.asString(), e);
                 return Mono.empty();
-            }).block();
+            });
     }
 
-    private InputStream zipMailboxesContent(Username username) throws Exception {
+    public InputStream zipMailboxesContent(Username username) throws IOException {
```

Comment:
Why putting it back to public? It's not accessed outside of this class, keep it private

Reply:
Sorry it was a mistake. i attempted to use this method in the test and forgot to change it back to private.